workflow/template2.osw

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
I'm not really sure what you're going for here. If someone tries to run this OSW, they will get errors and be confused. If this is actually meant to be another example template OSW file, then A) we should use more descriptive names to distinguish the multiple template OSWs and B) we should construct the OSWs so that they can run as is.

Reply:
A) Yes, we could rename them if there are 2+ template OSWs.

B) The problem with referencing an actual sample file is that it would overwrite it. Maybe that's ok? Suggestions?